packages/sx.js/src/clients/offchain/ethereum-sig/index.ts

```diff
@@ -0,0 +1,103 @@
+import { offchainGoerli } from '../../../offchainNetworks';
+import { domain, voteTypes } from './types';
+import type { Signer, TypedDataSigner, TypedDataField } from '@ethersproject/abstract-signer';
+import type { Vote, Envelope, SignatureData, EIP712VoteMessage, EIP712Message } from '../types';
+import type { OffchainNetworkConfig } from '../../../types';
+
+const SEQUENCER_URLS = {
+  1: 'https://seq.snapshot.org',
+  5: 'https://testnet.seq.snapshot.org'
+};
+
+type EthereumSigClientOpts = {
+  networkConfig?: OffchainNetworkConfig;
+  sequencerUrl?: string;
+};
+
+export class EthereumSig {
+  sequencerUrl: string;
+  networkConfig: OffchainNetworkConfig;
+
+  constructor(opts?: EthereumSigClientOpts) {
+    this.networkConfig = opts?.networkConfig || offchainGoerli;
+    this.sequencerUrl = opts?.sequencerUrl || SEQUENCER_URLS[this.networkConfig.eip712ChainId];
+  }
+
+  public async sign<T extends EIP712VoteMessage>(
+    signer: Signer & TypedDataSigner,
+    message: T,
+    types: Record<string, TypedDataField[]>
+  ): Promise<SignatureData> {
+    const address = await signer.getAddress();
+    const EIP712Message: EIP712Message = {
```

Comment:
```suggestion
    const message: EIP712Message = {
```

Right now it's using same name as the type, normal variable shouldn't start with capital letter as well.

Reply:
`message` already used by a function arg